sdk/tests/conformance2/uniforms/gl-uniform-arrays-sub-source.html

```diff
@@ -331,6 +340,16 @@
     }
   }
 
+  // Initialize uniforms to value 99 and verify it.
+  var initValues = new Array(typeInfo.numElementsPerType * 3);
+  for (var ii = 0; ii < typeInfo.numElementsPerType * 3; ++ii) {
```

Comment:
Could you please define a variable with some documentation and use it in place of these instances of "3" and above?